cupy/testing/helper.py

```diff
@@ -1098,6 +1100,9 @@ def test_func(self, *args, **kw):
                 try:
                     kw[name] = a
                     impl(self, *args, **kw)
+                    a = tuple(i - ndim for i in a)
```

Comment:
Conventionally we write as follows because building tuple from a list is faster than from a generator.
```suggestion
                    a = tuple([i - ndim for i in a])
```